Source/mmus2mid.c

```diff
@@ -302,8 +303,12 @@ int mmus2mid(UBYTE *mus, MIDI *mididata, UWORD division, int nocomp)
   signed char MUS2MIDchannel[MIDI_TRACKS];
 
   // copy the MUS header from the MUS buffer to the MUSh header structure
-
-  memcpy(&MUSh,mus,sizeof(MUSheader));
+	memcpy(MUSh.ID, mus, 4);
+	MUSh.ScoreLength = READ_INT16(&mus[4]);
```

Comment:
Thank you! Could you please keep the indentation identical to the original code?